src/bundle.js

```diff
@@ -0,0 +1,28 @@
+import virtual from '@rollup/plugin-virtual';
+import { rollup } from 'rollup';
+
+export async function bundle(input) {
+  try {
+    const bundle = await rollup({
+      input,
+      plugins: [
+        virtual({
+          'fastly:backend': `export const Backend = globalThis.Backend`,
+          'fastly:cache-override': `export const CacheOverride = globalThis.CacheOverride`,
+          'fastly:config-store': `export const ConfigStore = globalThis.ConfigStore`,
+          'fastly:dictionary': `export const Dictionary = globalThis.Dictionary`,
+          'fastly:env': `export const env = globalThis.fastly.env.get`,
+          'fastly:geolocation': `export const lookup = globalThis.fastly.getGeolocationForIpAddress`,
```

Comment:
Perhaps keep the name `getGeolocationForIpAddress`? Or something with `geo` in the name like `geoLookup`?